frontend/packages/knative-plugin/src/utils/create-eventsources-utils.ts

```diff
@@ -436,3 +457,24 @@ export const handleRedirect = (
   const redirectURL = perspectiveData.properties.getImportRedirectURL(project);
   history.push(redirectURL);
 };
+
+export const useGetKamelet = (kameletName: string, namespace: string) => {
+  const resource = useK8sWatchResources({
```

Comment:
as in this custom hook we are just making use of `useK8sWatchResources` , we may not need this and call useK8sWatchResources directly WDYT?